Unity/Assets/Scripts/WeaponsScripts/CollectionWeapons.cs

```diff
@@ -14,26 +14,45 @@ public class CollectionWeapons : MonoBehaviour
     /// Each element represents a different weapon (sword, axe, etc.) that can be activated.
```

Comment:
![image](https://github.com/user-attachments/assets/4d542338-fa81-48c5-a3c9-2ff99436f086)
Please add the nameSpace line

Reply:
I'll handle this